docs/source/working_with_aiida/troubleshooting.rst

```diff
@@ -15,16 +15,19 @@ Only warnings and errors are logged to the daemon log files, while info and debu
 messages are discarded.
 
 If you are experiencing a problem, you can change the default minimum logging
-level of AiiDA messages (and circus messages -- circus is the library that we
-use to manage the daemon process) using, on the command line, the two
-following commands::
+level of AiiDA messages::
 
-  verdi config logging.circus_loglevel DEBUG
   verdi config logging.aiida_loglevel DEBUG
 
+You might also be interested in circus log messages (circus is the library that manages AiiDA processes) but most often it is used by AiiDA developers::
```

Comment:
"that manages AiiDA processes" this might be misunderstood to mean instances of the `Process` class, which is not true. The `circus` library is the daemonizer that manages the daemon runners